app/enterprise/2.1.x/deployment/upgrades/migrations.md

```diff
@@ -69,44 +69,44 @@ affect your current installation.
-that while the migration is in process, you have two Kong clusters with different
-versions running that are sharing the same database. This is sometimes referred
-to as the
-[blue-green migration model](https://en.wikipedia.org/wiki/Blue-green_deployment).
-
-The migrations are designed so that there is no need to fully copy
-the data. The new version of {{site.ee_product_name}} is able to use the data as it
-is migrated, and the old
-Kong cluster keeps working until it is finally time to decommission it. For this
-reason, the full migration is split into two commands:
-
-- `kong migrations up`: performs only non-destructive operations
-- `kong migrations finish`: puts the database in the final expected state (DB-less
-  mode is not supported in {{site.ee_product_name}})
-
-1. Download 2.1.x, and configure it to point to the same datastore as your old
-   1.5.x (or 2.1.x-beta) cluster.
-2. Run `kong migrations up`.
-3. After that finishes running, both the old (1.5) and new (2.1) clusters can
-   now run simultaneously on the same datastore. Start provisioning 2.1 nodes,
-   but do _not_ use their Admin API yet.
+Due to internal changes, the table schemas used by Kong 2.1.x are incompatible
+with those used by Kong 1.5.x. Migrating using the usual commands `kong
+migrations up` and `kong migrations finish` will require a small window of
+downtime, since the old and new versions cannot use the database at the same
+time.
+
+Alternatively, to keep your previous version fully operational while the
+new one initializes, you will need to cache the entities to prevent database
+access, as described below:
+
+1. Update the Kong 1.5.x cluster and set the
+   [`db_cache_warmup_entities`](https://docs.konghq.com/enterprise/1.5.x/property-reference/#db_cache_warmup_entities)
+   configuration value. Specify all the entities Kong has configured.
+2. [Reload](https://docs.konghq.com/1.5.x/cli/#kong-reload) the Kong 1.5.x nodes
+   to pick up this new configuration value:
+
+   ```shell
+   $ kong reload [-c configuration_file]
+   ```
 
-   **Important:** If you need to make Admin API requests,
-   these should be made to the old cluster's nodes. This prevents
-   the new cluster from generating data that is not understood by the old
-   cluster.
+   ** Note:** Monitor the `pg_stat_statements` on the Postgres server to ensure
+   no additional entities need to be warmed in the cache (added to the list in
+   the configuration).
 
-4. Gradually divert traffic away from your old nodes, and redirect traffic to
+3. Download 2.1.x, and configure it to point to the same datastore as your old
+   1.5.x (or 2.1.x-beta) cluster.
+4. Run `kong migrations up` on the 2.1.x cluster.
+5. Run `kong migrations finish` on the 2.1.x cluster. From this point onward, it
+   is no longer possible to start nodes in the old 1.5 (or 2.1.x-beta) cluster
+   or access its admin API. Run this command _only_ when you are confident that
```

Comment:
I wasn't completely sure if users could no longer access the admin API on the 1.5 cluster - do changes to the 1.5 admin API get stored in the database?

Also, looking at this, it probably needs to be moved to `4.` if the `migrations up` command would interfere with the the ability of the 1.5 cluster to store admin API changes.

Reply:
> I wasn't completely sure if users could no longer access the admin API on the 1.5 cluster - do changes to the 1.5 admin API get stored in the database?
> 
> Also, looking at this, it probably needs to be moved to `4.` if the `migrations up` command would interfere with the the ability of the 1.5 cluster to store admin API changes.

@tyler-ball thank you for the PR. I cannot answer your question; an engineer should be able to confirm and also review and test this PR. Any chance you could assist with the 2.2 to 2.3 migration docs/testing as well? @RobSerafini who can assist docs with this? cc: @GayleNeumann